static/app/views/performance/trends/utils.tsx

```diff
@@ -261,7 +262,17 @@ export function modifyTrendView(
       trendParameter.column
     );
   }
-  trendView.query = getLimitTransactionItems(trendView.query);
+
+  if (!organization.features.includes('performance-new-trends')) {
+    trendView.query = getLimitTransactionItems(trendView.query);
+  } else {
+    const query = new MutableSearch(trendView.query);
+    // remove metrics-incompatible filters
+    if (query.hasFilter('transaction.duration')) {
+      query.removeFilter('transaction.duration');
+    }
```

Comment:
Doing metrics incompatibility the frontend probably won't work long term, is this just for a common use case?

Reply:
yeah I need to chart with Albert about handling metric-incompatible filters. maybe we'll limit the search bar to metrics-supported filters